internal/authz/spicedb/spicedb.go

```diff
@@ -72,7 +72,7 @@ func (p Permission) AddRelation(ctx context.Context, relation model.Relation) er
 	request := &pb.WriteRelationshipsRequest{
 		Updates: []*pb.RelationshipUpdate{
 			{
-				Operation:    pb.RelationshipUpdate_OPERATION_CREATE,
+				Operation:    pb.RelationshipUpdate_OPERATION_TOUCH,
```

Comment:
why touch?

Reply:
TOUCH is upsert, with CREATE I was getting error for duplicate entries.
https://buf.build/authzed/api/file/main/authzed/api/v1/core.proto#L65